Apps/WebClient/src/ClientApp/src/models/userComment.ts

```diff
@@ -1,19 +1,19 @@
-export default interface UserComment {
+export default class UserComment {
```

Comment:
this looks like a very hacky solution :(

I think I rather see the warning and wait until we understand why it's showing instead of making it

Reply:
Sounds good!